dynaconf/loaders/vault_loader.py

```diff
@@ -130,3 +130,17 @@ def write(obj, data=None, **kwargs):
     path = "/".join([obj.VAULT_PATH_FOR_DYNACONF, obj.current_env.lower()])
     client.secrets.kv.create_or_update_secret(path, secret=data)
     load(obj)
+
+
+def load_keys(obj, path):
```

Comment:
Lets make `path` optional? if not passed then it will be taken from object itself?
```suggestion
def list_envs(obj, path=None):
    ...
    path = path or obj.get('VAULT_PATH_FOR_DYNACONF')
```

Reply:
Got it. I will check how it behaves.